package.json

```diff
@@ -43,9 +43,10 @@
         "storybook": "run-p watch build:watch storybook:start",
         "storybook:start": "start-storybook -p 6006",
         "storybook:build": "build-storybook -c .storybook -o ./documentation/dist/storybook",
+        "docs:analyze": "wca analyze 'packages/*/src/**/*.ts' --format json --outFile documentation/custom-elements.json",
         "docs:build": "yarn build && yarn docs:ci",
-        "docs:ci": "gulp docsCompile && yarn storybook:build",
-        "docs:start": "gulp docsWatchCompile",
+        "docs:ci": "yarn docs:analyze && gulp docsCompile && yarn storybook:build",
```

Comment:
This is one time, so right now you have to manually run analyze for updates...

wca doesn't support a `live` mode, but we could work to add this via scripting on our end if people felt strongly about it.